code/wdb/datastripe.q

```diff
@@ -59,6 +59,32 @@ initdatastripe:{
 
 \d .ds
 
+symlink:{
+    /- function to create HDB sym file and symlink to this sym file at start up
+    /- create HDB sym file
+    symdir:`$raze string .wdb.hdbdir,"/sym";
```

Comment:
I think the variable name here is misleading, as we're not creating a directory

you also shouldn't need the raze

```suggestion
    symdir:`$string[.wdb.hdbdir],"/sym";
```

or possibly can just use `sv` https://code.kx.com/q/ref/sv/#filepath-components

```suggestion
    symdir:` sv (wdb.hdbdir;`sym);
```

Reply:
good point, I've renamed the variable to `sympath` and went for the `sv` method to keep it consistent with other path creations in this file